### PR TITLE
search widget place holder text and nls reference

### PR DESCRIPTION
**Pull request by esoekianto**
issue #529

### CODE REVIEW

**Comment by driskull** [comment]
@kellyhutchins would you want this done for your other templates too?

File: js/SearchSources.js [hunk 1]
```diff
@@ -139,7 +140,7 @@ define(["dojo/_base/declare", "dojo/_base/lang", "dojo/_base/array", "dojo/_base
             source.searchFields = [searchLayer.field.name];
             source.displayField = searchLayer.field.name;
             source.outFields = ["*"];
-            source.placeholder = searchOptions.hintText;
+            source.placeholder = nls.user.find;
```
Comment (driskull) [inline js/SearchSources.js]: What about:

```
source.placeholder = searchOptions.hintText || nls.user.find;
```

**Comment by kellyhutchins** [comment]
@driskull  My templates don't have the hint text defined in nls. I'm pretty sure we just use the default placeholder text from the search widget if not specified.

File: js/SearchSources.js (hunk 1 above)
Comment (esoekianto) [inline js/SearchSources.js]: I will make this changes tomorrow

**Pull request merged**